CHANGELOG.md

```diff
@@ -0,0 +1,246 @@
+# Change Log for AWS IoT Embedded C SDK
+
+## 202009.00 (September 14, 2020)
+
+- This release introduces re-designed MQTT client library, JSON parser library and the AWS IoT Shadow client library.
+- The AWS IoT Jobs service client library is not part of this release. It will be part of a future release.
+- The MQTT, JSON and AWS IoT Shadow service client libraries have been moved to the following individual repositories:
+   - [FreeRTOS/coreMQTT](https://github.com/FreeRTOS/coreMQTT) for MQTT client library
+   - [FreeRTOS/coreJSON](https://github.com/FreeRTOS/JSON) for JSON parser library
+   - [aws/device-shadow-for-aws-iot-embedded-sdk](https://github.com/aws/device-shadow-for-aws-iot-embedded-sdk) for AWS IoT Shadow service client library.
+- The above library folders in `libraries/` folder has been replaced with submodule to the individual repositories.
+- Commit [6f88ba7](https://github.com/aws/aws-iot-device-sdk-embedded-C/commit/6f88ba7e5471b550f9b9c0fb9bed7ef6e1230caf) has been tagged as [v3.1.0](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.1.0) SDK. The commit was part of the `master` branch following
+the commit tree of the [v3.0.1](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.0.1) release. [v3.1.0](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.1.0) is **NOT** a released
+version.  
```

Comment:
what does that mean? its tagged, why is not released? 

If I am a customer reading it, what do I take away from it? Should I use v3.1.0 or not?

Reply:
"Not released" raises these questions. 
I believe we should call out v3.1 on a separate line saying that is the last V3 commit as of 09/2020. 
I believe by 'not released' we mean no special testing has been performed on the tip of the last master, but we have tagged it for the reference.

Reply:
Have created a separate section for `v3.1.0`